pyInterface/test_MC.sh

```diff
@@ -0,0 +1,153 @@
+
+cp "${ROOTPWA}/userAnalysisWorkspace/3pi.--+/generator_noBeamSimulation.conf" "./"
+cp "${ROOTPWA}/userAnalysisWorkspace/3pi.--+/keyfiles/GenerateKeyfiles.sh" "./"
+cp "${ROOTPWA}/userAnalysisWorkspace/3pi.--+/keyfiles/template.key" "./"
+cp "${ROOTPWA}/userAnalysisWorkspace/3pi.--+/keyfiles/wavelist.compass.2008.88waves" "./"
+cp "${ROOTPWA}/pyInterface/rootpwa.config" "./"
+cp "${ROOTPWA}/pyInterface/mcTest/reference_fit/bin65_c2pap_bestfits_converged_MASS_1800_1820_N45340.root.example" "./reference_fit/bin65_c2pap_bestfits_converged_MASS_1800_1820_N45340.root"
+
+#-- END PREPARATION --#
+
+### BEGIN MONTE CARLO GENERATION ###
+
+# generate phase space data
+echo "Generating phase space ..."
+${ROOTPWA}/pyInterface/genpw.py -n $NMB_PS_EVENTS -p "${PARTICLE_DATA_TABLE}" -M $MASS -B $BINWIDTH "./generator_noBeamSimulation.conf" -o "./data/phase_space_MASS_${MASS}-$((MASS+BINWIDTH))_N_${NMB_PS_EVENTS}.root"
+
+# generate the keyfiles
+echo "Generating keyfiles ..."
+DESTINATION_DIR="${TESTDIR}/keyfiles"
+export PARTICLE_DATA_TABLE
+export DESTINATION_DIR
+./GenerateKeyfiles.sh 
+
+# remove some parameterizations
+rm ${TESTDIR}/keyfiles/*[f0_980_0bw=[*.key
+rm ${TESTDIR}/keyfiles/*[f0_980_0=[*.key
+
+# create first file manager
+echo "Create first file manager ..."
+${ROOTPWA}/pyInterface/userInterface/createFileManager.py
+
+# calculate amplitudes for generated data
+echo "Calculate amplitudes for generated data ..."
+${ROOTPWA}/pyInterface/userInterface/calcAmplitudes.py -e generated
+
+# calculate integrals for generated data
+echo "Calculate integrals for generated data ..."
+${ROOTPWA}/pyInterface/userInterface/calcIntegrals.py -e generated
+
+# generate weighted MC pseudo data
+echo "Generate weighted MC pseudo data ..."
+${ROOTPWA}/pyInterface/genPseudoData.py "./generator_noBeamSimulation.conf" "${TESTDIR}/reference_fit/bin65_c2pap_bestfits_converged_MASS_1800_1820_N45340.root" "./ints/integral_binID-0_2.root" "./weighted_mc_data/weighted_pseudoData_MASS_${MASS}-$((MASS+BINWIDTH))_N_${NMB_PSEUDO_EVENTS}.root" -s 654321 -n ${NMB_PSEUDO_EVENTS} -M ${MASS} -B ${BINWIDTH} 
+
+# deweight the pseudo data
+echo "Deweight pseudo data ..."
+${ROOTPWA}/pyInterface/deWeight.py "./weighted_mc_data/weighted_pseudoData_MASS_${MASS}-$((MASS+BINWIDTH))_N_${NMB_PSEUDO_EVENTS}.root" "./data/pseudoData_MASS_${MASS}-$((MASS+BINWIDTH))_N_${NMB_PSEUDO_EVENTS}.root"
+
+# backup old file manager ...
+echo "Move first file manager (backup) ..."
+mv ./fileManager.pkl ./fileManager.pkl.old
+
+# ... create a new file manager
+echo "Create second file manager ..."
+${ROOTPWA}/pyInterface/userInterface/createFileManager.py
+
+# calculate amplitudes for 'real' (= MC) data
+echo "Calculate amplitudes for 'real' (= MC) data ..."
+${ROOTPWA}/pyInterface/userInterface/calcAmplitudes.py -e real
+
+# 'fake' accepeted integrals -> acceptance = 1
+echo "'Fake' accepted integrals ..."
+ln -s "${TESTDIR}/ints/integral_binID-0_2.root" "${TESTDIR}/ints/integral_binID-0_3.root"
```

Comment:
the cleaner approach would be to add a `--no-acceptance` option (or similar) to `pwaFit.py` and `pwaNloptFit.py`. (probably not for this pull request, but we need to keep this in mind.)